domain/model/errors/errors.go

```diff
@@ -36,4 +36,7 @@ const (
 	// a user on a particular model cannot be found. This does not mean the user
 	// does not exist.
 	UserNotFoundOnModel = errors.ConstError("user not found on model")
+
+	// InvalidModelStatus describes an error that occurs when an invalid model status to be set.
```

Comment:
Should read more like: InvalidModelStatus describes an error that occurs when a status is being set on a model that is not in the allowed set of status values.